### DISCUSSION TITLE
Different page depending on the User-Agent

### DISCUSSION THREAD

**Discussion by blumgart** [nuxt/nuxt#22211] [Questions]
Hello, is it possible to show a prerendered page depending on the User-Agent? It means that if the User-Agent: "XXX" shows one prerendered page, if the User-Agent: "YYY" shows another SSR page, but at the same URL?

Or, depending on the user agent, show the SSR and SPA?

**Comment by TheAlexLichter** [comment]
Yes, this is possible when using SSR in general:
Define a nitro middleware, do your checks and then set:

```ts
export default defineEventHandler((event) => {
  if (YOUR_CONDITION_HERE) {
    event.context.nuxt = event.context.nuxt || {}
    event.context.nuxt.noSSR = true
  }
})
```
